package.json

```diff
@@ -99,6 +99,7 @@
 		"equivalent-key-map": "0.2.2",
 		"esbuild": "0.18.20",
 		"escape-html": "1.0.3",
+		"eslint": "8.3.0",
```

Comment:
This is the same problem. Changing a string doesn't require adding new project-wide dependencies.

The only change in this PR should be in the `packages/block-library/src/navigation/edit/index.js` file and for the `submenuAccessibilityNotice` as the issue suggests.